easycv/models/segmentation/heads/pixel_decoder.py

```
@@ -11,7 +11,7 @@
 from .transformer_decoder import PositionEmbeddingSine, _get_activation_fn
 
 try:
-    from thirdparty.deformable_transformer.modules import MSDeformAttn
+    from thirdparty.msdeformattn.modules import MSDeformAttn
 except:
     pass
```

try.... except....pass with no logging or raise 是不推荐使用的用法，希望这里还是能加一个报错或者说明影响的范围，比如我这种不明白为什么要import 这个东西的读者 会质疑，如果可以try except pass with no logging ，那为什么不直接删掉。

---

> try.... except....pass with no logging or raise 是不推荐使用的用法，希望这里还是能加一个报错或者说明影响的范围，比如我这种不明白为什么要import 这个东西的读者 会质疑，如果可以try except pass with no logging ，那为什么不直接删掉。

这里之前贺弘可能因为msdeformattn不是所有算法必须的，保证不会报错吧

---

done

---

delay the following import to where MSDeformAttn is used but not at the top of file

from thirdparty.msdeformattn.modules import MSDeformAttn

---

done